files/storage_usage.py

```
@@ -0,0 +1,206 @@
+    Media.objects.filter(pk=media.pk).update(storage_usage_bytes=used_bytes)
+    invalidate_storage_usage_cache(user_id=media.user_id)
+    return used_bytes
+
+
+def schedule_refresh_media_storage_usage(media_or_id):
+    media_id = getattr(media_or_id, "id", media_or_id)
+
+    def refresh():
+        try:
+            refresh_media_storage_usage(media_id)
+        except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)